src/app/application/components/app-collection/AppCollection.vue

```diff
@@ -216,8 +216,8 @@ function getRowAttributes(row: Row): Record<string, string> {
 const click = (e: MouseEvent) => {
   const $tr = (e.target as HTMLElement).closest('tr')
   if ($tr) {
-    const $a = $tr.querySelector('a')
-    if ($a !== null) {
+    const $a: HTMLAnchorElement | null = $tr.querySelector('td:first-child a')
+    if ($a !== null && $a.closest('tr, li') === $tr) {
       $a.click()
     }
   }
```

Comment:
Ah I meant to say, I think moving forwards we should use a `data-something` attribute to be able to 'tag' the link that gets clicked on row click. So then this would just be `const $a = $tr.querySelector('a[data-something]')`. 

Thing is that would have meant changing an awful lot in this PR, so this is a temporary measure so that we can use AppCollection for the table here.

I can follow up once this goes in with a PR that does the above 'tagging' alternative.

Reply:
P.S.

> It looks to me that selecting the "row click link" by a dedicated selector on that link would be better because it would be unambiguous.

This ☝️

Reply:
> I can follow up once this goes in with a PR that does the above 'tagging' alternative.

lmk on this one and I can get on with that work whilst we are looking at this PR